stories/point-group.stories.jsx

```diff
@@ -256,32 +280,38 @@ export const editMultiplePointsRemovePoint3OnDone = {
                 description:
                   'Point 2 Description, Point 2 Description, Point 2 Description, Point 2 Description, Point 2 Description, Point 2 Description, Point 2 Description, ',
                 groupedPoints: [],
-                user: {
-                  dob: '1980-10-20T00:00:00.000Z',
-                  state: 'GA',
-                  party: 'Independent',
+                demInfo: {
```

Comment:
There are lots of changes and it's going to take some time to review.  Bot one things is that the demoInfo property should not have user, it should just be:
```
demInfo: {
                    dob: '1980-10-20T00:00:00.000Z',
                    state: 'GA',
                    party: 'Independent',
}
```